src/components/help/contact-forms/shared/SupportForm_AttachmentUploader.tsx

```diff
@@ -0,0 +1,252 @@
+// import { CreateTicketInput } from "@3rdweb-sdk/react/hooks/useApi";
```

Comment:
if its not ready just delete the file for now

Reply:
thanks!